app/Models/Application.php

```
@@ -1555,40 +1556,185 @@ protected function buildGitCheckoutCommand($target): string
             return false;
         }
-        $watch_paths = collect(explode("\n", $this->watch_paths))
-            ->map(function (string $path): string {
-                return trim($path);
-            })
-            ->filter(function (string $path): bool {
-                return strlen($path) > 0;
-            });
+        $this->watch_paths = $this->parseWatchPaths($this->watch_paths);
+        $this->save();
+        $watch_paths = collect(explode("\n", $this->watch_paths));
```

PHPMD (2.15.0)

The variable $watch_paths is not named in camelCase. (undefined)

(CamelCaseVariableName)